apps/meteor/app/e2e/client/rocketchat.e2e.ts

```diff
@@ -418,15 +418,31 @@ class E2E extends Emitter {
 			return message;
 		}
 
-		const data = await e2eRoom.decrypt(message.msg);
+		const attachmentDescription = message.attachments?.[0]?.description;
```

Comment:
We don't support multiple files yet, but we will support that soon. Do you think it makes sense to prepare this to support that?